port/libusb/Makefile

```diff
@@ -8,7 +8,7 @@ COMMON += btstack_audio_portaudio.c btstack_chipset_zephyr.c btstack_chipset_rea
 
 include ${BTSTACK_ROOT}/example/Makefile.inc
 
-CFLAGS  += -g -std=c99 -Wall -Wmissing-prototypes -Wstrict-prototypes -Wshadow -Wunused-parameter -Wredundant-decls -Wsign-compare -Wswitch-default
+CFLAGS  += -g -std=gnu99 -Wall -Wmissing-prototypes -Wstrict-prototypes -Wshadow -Wunused-parameter -Wredundant-decls -Wsign-compare -Wswitch-default
```

Comment:
Why gnu99 instead of c99?

Reply:
If c99 is used, container_of() will result in an error during compilation. 

../../chipset/realtek/btstack_chipset_realtek.c:373:20: error: expected declaration specifiers or ‘...’ before ‘(’ token
     const typeof(  ((type *)0)->member) *__mptr = (ptr);    \
                    ^
../../chipset/realtek/btstack_chipset_realtek.c:377:5: note: in expansion of macro ‘container_of’
     container_of(ptr, type, member)
     ^~~~~~~~~~~~
../../chipset/realtek/btstack_chipset_realtek.c:962:23: note: in expansion of macro ‘list_entry’
                 tmp = list_entry(pos, struct patch_node, list);
                       ^~~~~~~~~~
../../chipset/realtek/btstack_chipset_realtek.c:374:23: error: ‘__mptr’ undeclared (first use in this function); did you mean ‘fw_ptr’?
     (type *)( (char *)__mptr - offsetof(type,member) );})